docs/files/integration.rst

```diff
@@ -0,0 +1,162 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+.. _integration:
+
+Integrating Files
+=================
+
+Invenio provides a bundle of modules to accommodate all around needs about
+file management. You can install all the necessary modules at once, or due to
+invenio modular architecture you can hand pick only those you need.
+
+In case you have scaffolded your invenio application with
+`cookiecutter-invenio-instance <https://github.com/inveniosoftware/cookiecutter-invenio-instance>`_
+files bundle is already a part of your solution, the only step you need to
```

Comment:
This assumes that the reader is starting from cookiecutter with Invenio 3.2. If they created their project with Invenio 3.1 then upgraded to 3.2 this is not true. Should we mention this only applies if you started your project from 3.2?